project/project_ps.go

```diff
@@ -1,13 +1,17 @@
 package project
 
-import (
-	"golang.org/x/net/context"
-)
+import "golang.org/x/net/context"
 
 // Ps list containers for the specified services.
 func (p *Project) Ps(ctx context.Context, services ...string) (InfoSet, error) {
 	allInfo := InfoSet{}
-	for _, name := range p.ServiceConfigs.Keys() {
+
+	if services == nil {
```

Comment:
Tests are correctly failing. This line should be `if len(services) == 0`. Otherwise it does not display any services from the CLI (because `services ...string` is not `nil`, it's an empty array).